woocommerce/wc-admin
# Add task list and activity panel E2E tests

Added E2E tests for hiding and showing the task list, and some simple tests for the activity panels.
I made use of `@woocommerce/api` and `@woocommerce/e2e-utils` for creating products and orders on the fly.

No changelog as it is in the package.

### Detailed test instructions:

- Run E2E tests locally `npm install && npx wc-e2e docker:up` and then `npx wc-e2e test:e2e`
- Check if the github action passed correctly
